packages/blocks/src/page-block/default/selection-manager.ts

```diff
@@ -71,7 +71,12 @@ function createSelectionRect(
   return new DOMRect(left, top, width, height);
 }
 
-type PageSelectionType = 'native' | 'block' | 'none' | 'embed';
+type PageSelectionType =
+  | 'native'
+  | 'native_collapsed'
```

Comment:
1. This may overlap with other types since I guess a `native_collapsed` selection type is also `native`. This may lead to confusion and tricky bugs.
2. We don't use `snake_case`

Reply:
@SaikaSakura